Allow emitting mutable buffer names in schema

JacobSzwejbka opened the pull request:
Summary: part 1 of letting people query tensors by names in runtime.

Differential Revision: D72579501




cc @angelayi

lucylq reviewed (approved):
Looks good! What is this intended to be used for, btw?

JacobSzwejbka committed:
Allow emitting mutable buffer names in schema (#9935)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/9935

part 1 of letting people query tensors by names in runtime.

Reviewed By: tarun292, lucylq

Differential Revision: D72579501
